tests/wallet/test_singleton_lifecycle.py

```diff
@@ -1,144 +0,0 @@
-from __future__ import annotations
-
-from typing import List, Tuple
```

Comment:
did you mean to delete this? It seems unrelated to the main rationale for this change

Reply:
Yeah, the test was failing and when I took a look at it, I realized it was an unfinished, obsolete test whose functionality is covered elsewhere.  Probably should have been its own PR but I figured it was a simple enough deletion I could sneak it in, my bad.